master/buildbot/steps/source/repo.py

```diff
@@ -153,6 +154,9 @@ def __init__(self,
         @type depth: integer
         @param depth: optional depth parameter to repo init.
                           If specified, create a shallow clone with given depth.
+
+        @type syncQuietly: bool.
```

Comment:
please add syncQuietly to the allowed words in common/code_spelling_ignore_words.txt

Reply:
@tardyp I have made this change and rebased the PR